.github/workflows/encryption-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Encryption Tests 
+
+on:
+  push
+  #workflow_dispatch: {}
+
+permissions:
+  contents: write
+  pull-requests: write
+  id-token: write
+
+jobs:
+  run-tests:
+    permissions:
+      # required for all workflows
+      security-events: write
+      id-token: write
+      contents: write
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [20]
+        os: [ubuntu-latest]
+        mongodb: [8.0.0]
```

Comment:
A testing matrix isn't actually necessary when you're only testing on one variant of each matrix axis (we're always testing node 20, ubuntu latest, server 8.0) - so we can just hard code these values into the script wherever they're needed.

Reply:
Removed it!